indico-dev/worker/indico.conf

```diff
@@ -0,0 +1,59 @@
+import ast
+import os
+
+
+SQLALCHEMY_DATABASE_URI = "postgresql://{}:{}@{}:{}/{}".format(os.environ['PGUSER'], os.environ['PGPASSWORD'],
```

Comment:
Just my two cents on this: It might be better to use an F-String for this (readability).

```python
SQLALCHEMY_DATABASE_URI = f"postgresql://{os.environ['PGUSER']}:{os.environ['PGPASSWORD']}@{os.environ['PGHOST']}:{os.environ['PGPORT']}/{os.environ['PGDATABASE']}"
```

Of course, you can line-break this for PEP8 conformity.

Reply:
I had version working nicely with a volume, but I ditched it because the setup was a bit complicated. I could add it back in so you can see what it looks like